### DISCUSSION TITLE
Unable to commit: Issue

### DISCUSSION THREAD

**Discussion by JuanmaMN** [community/community#22464] [Projects and Issues]
<p>Hi,</p>
<p>I seem to have an issue when I try to commit a change within Github across all my repositories.  I can commit from R studio as usual but when I want to edit any page within Github, the “commit changes” button doesn’t let me click on it.</p>
<p>Hope you can help. This issue has been happening since last Tuesday.</p>
<p>Please let me know if you need further information from my side.</p>
<p>Any help would be much appreciated.<br>
Juanma</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>what is your browser ?</p>
<p>it seems to be not working on the user end,</p>
<p>this is another one posted with a solution, you may want to check this out</p>
<aside class="quote" data-post="1" data-topic="234480">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/cannot-preview-or-commit-edits/234480">Cannot preview or commit edits</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Need help with your GitHub account or the core features of GitHub? This is the place to look for answers.">GitHub Help</span></a>
  </div>
  <blockquote>
    I’ve been editing files on GitHub for years.  Suddenly, I cannot preview or ‘commit’ an edit.  (Also, I cannot name a successor in my settings.)  I think there are other things wrong, too; but something has changed and I would dearly like to get it fixed.  I’ve been beating my head against this for hours.
  </blockquote>
</aside>

**Comment by JuanmaMN** [comment]
<p>Thanks a lot. I actually tried on Chrome and Firefox and had the issue. It seems to be working now. Thanks for sharing the comment, I will consider the point raised there.</p>
<p>I was surprised because I’ve been using Github for years and it’s never happened to me. Glad that it’s working now.</p>
<p>I appreciate your reply.</p>
